ble-cliapp/source/Commands/parameters/AdvDataBuilder.cpp

```diff
@@ -15,7 +15,7 @@
 #include "Serialization/GapSerializer.h"
 #include "Serialization/GapAdvertisingDataSerializer.h"
 #include "Serialization/Hex.h"
-#include "Common.h"
+#include "../Common.h"
```

Comment:
accidental change?

Reply:
Not at all; that is why it ended up in its individual commit. Compilation was broken on mbed-os `main` due to the inclusion of a `Common.h` file.